app/src/main/java/net/geekinpurple/www/appofourown/WorkInstance.java

```diff
@@ -63,4 +66,23 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
         return super.onOptionsItemSelected(item);
     }
+
```

Comment:
Dead code.

I know you're probably planning on using it later, but for now it's dead code.